### PR TITLE
[11-03-2023] Update VDL context repo.

### PR DESCRIPTION
**Pull request by kezike**
- Update VDL context content.
- Update Karma configuration.
- Update GitHub Workflow.
- Update NPM modules.
- Update licenses.
- Update tests.

### CODE REVIEW

**Review by dlongley — COMMENTED** [review]
@davidlehn,

I assume we want some ESM changes here too?

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -26,12 +26,12 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [18.x, 20.x]
```
Comment (davidlehn) [inline .github/workflows/main.yaml:29]: Karma only needs to run once using a recent node version:
```suggestion
        node-version: [20.x]
```

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -47,11 +47,11 @@ jobs:
...
```
Comment (davidlehn) [inline .github/workflows/main.yaml:50]: Linting only needs to run once using a recent node version:
```suggestion
        node-version: [20.x]
```

File: README.md [hunk 3]
```diff
@@ -19,36 +20,53 @@
...
+
+// Default context data
+CONTEXT
+// -> full context object
 
 // Codec term map value for CBOR-LD
-exampleCtx.constants.CBORLD_CODEC_VALUE
-// 0x0..
+constants.CBORLD_VALUE
+// -> 0x20
```
Comment (davidlehn) [inline README.md:66]: This is not arbitrary.  Show as zero or register a new one and fix constants file.  See https://github.com/digitalbazaar/vdl-context/issues/6.

**Comment by kezike** [comment]
> @davidlehn,
> 
> I assume we want some ESM changes here too?

@dlongley Done ✅

**Comment by kezike** [comment]
@davidlehn I noticed that the CBOR-LD discussion was revisited in the OpenCred Slack channel. Do we have an idea about what this value will be for the base VDL vocab?

**Review by davidlehn — APPROVED** [review]

**Pull request merged**